bay_platform_dependencies.install

```diff
@@ -18,3 +18,46 @@ function bay_platform_dependencies_install() {
 function bay_platform_dependencies_uninstall() {
   \Drupal::messenger()->addStatus(__FUNCTION__);
 }
+
+/**
+ * Update section purger plugin to sectionbundled
+ */
+function bay_platform_dependencies_update_8002() {
```

Comment:
@FleetAdmiralButter I have incremented the number to 8002 because I noticed you have a PR open with 8001.
https://github.com/dpc-sdp/bay_platform_dependencies/pull/2/files#diff-5ec226c78362e9520360495a7bb19a399b2246907d5776ef6c22dd5bbcdb8770R25
Once that PR is approved/merged, I will merge the changes into this PR.